pcdsdevices/sample_delivery.py

```diff
@@ -0,0 +1,440 @@
+"""
+Module for the Sample Delivery System and related devices.
+"""
+from ophyd.device import Component as Cpt
+from ophyd.device import Device
+from ophyd.signal import EpicsSignal
+
+from .component import UnrelatedComponent as UCpt
+from .signal import PytmcSignal
+
+
+class ViciValve(Device):
+    """
+    A Vici Valve as used in the SDS Selector.
+
+    Parameters
+    ----------
+    prefix : str
+        The base PV for the Selector.
+
+    name : str
+        A name for the device.
+    """
+
+    req_pos = Cpt(PytmcSignal, ':ReqPos', io='io', kind='normal')
+    curr_pos = Cpt(PytmcSignal, ':CurrentPos', io='i', kind='normal')
+
+
+class Selector(Device):
+    """
+    A Selector for the sample delivery system.
+
+    Parameters
+    ----------
+    prefix : str
+        The base PV for the Selector.
+
+    name : str
+        A name for the device.
+
+    names_button_prefix : str
+
+    couple_button_prefix : str
+
+    names1_prefix : str
+
+    names2_prefix : str
+    """
+
+    status = Cpt(PytmcSignal, ':IO:SyncUnitOK', io='i', kind='normal')
+
+    sampleFM_flow = Cpt(PytmcSignal, ':SampleFM:Flow', io='i', kind='normal')
+    sampleFM_state = Cpt(PytmcSignal, ':SampleFM:State', io='i',
+                         kind='normal')
```

Comment:
what about the flowrates?  e.g. `:SampleFlow`, `:LiquidSheathFlow`, `:GasSheathFlow` that appear in `ST_SelectorM3`

Reply:
Honestly, I don't understand why both `:SampleFM:Flow` and `:SampleFlow` exist

Reply:
![image](https://user-images.githubusercontent.com/25809004/85335536-06c4b100-b492-11ea-9e62-43658a5aa81b.png)
Yeah, I'm a little confused by the stack here

Reply:
@slacAWallace, care to comment?